servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -192,6 +201,7 @@ public void start(boolean isLeader) throws IOException {
   @Override
   public void stop() throws IOException {
     super.stop();
+    mTTLCheckerService.cancel(true);
```

Comment:
I think you should check to see if this is null first. If the master was in standby mode, it would be null.